libs/components/organisms/src/lib/biomarker-table/biomarker-table.component.scss

```diff
@@ -9,10 +9,10 @@
   }
 
   .table {
-    overflow-x: auto;
-    overflow-y: auto;
     min-width: auto;
+    min-height: auto;
     font-size: 0.875rem;
+    position: absolute;
```

Comment:
This seems sketchy...

Reply:
If we dont use absolute positioning then the scroll bar on the right hand side, and bottom scroll bar will move to the end of the screen in their respective directions.